components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -21,39 +21,45 @@
   <div *ngIf="serviceGroupsId && (serviceGroupsStatus$ | async) === 'loadingSuccess'">
     <div class="status-filter-bar">
       <chef-status-filter-group [value]="selectedHealth" lean>
-        <chef-option class="filter general" value="total" (click)="updateHealthFilter('total')" (keyup.enter)="updateHealthFilter('total')" selected>
+        <chef-option class="filter general" value="total" (click)="updateHealthFilter(allowedStatus.TOTAL)" (keyup.enter)="updateHealthFilter(allowedStatus.TOTAL)" selected>
           <chef-icon class="filter-icon">grain</chef-icon>
           <div class="filter-label">Total</div>
```

Comment:
Contrast this with the change above, though just food for thought--you could also do this:
```suggestion
          <div class="filter-label">{{allowedStatus.TOTAL | capitalize}}</div>
```
But I am not really suggesting this; here "Total" is pure presentation so to me is fine as is, whereas on L24 that is code so should be using `allowedStatus.TOTAL` as you have done.

Reply:
food for thought, thanks for pointing it out